apps/nextjs/.storybook/public/mockServiceWorker.js

```diff
@@ -8,133 +8,133 @@
  * - Please do NOT serve this file on production.
  */
 
-const PACKAGE_VERSION = '2.6.5'
-const INTEGRITY_CHECKSUM = 'ca7800994cc8bfb5eb961e037c877074'
-const IS_MOCKED_RESPONSE = Symbol('isMockedResponse')
-const activeClientIds = new Set()
+const PACKAGE_VERSION = "2.6.5";
```

Comment:
These quote changes seem to be automated when saving the file

Reply:
This file is autogenerated so I assume that we shouldn't be modifying or linting it?

Reply:
Thanks - I've removed that an added it to ignore config